### PR TITLE
Added content on  landing pages

### LINKED ISSUE #670: Add landing pages to docs
Some of our landing pages have no content, they should have some content.

### PR DESCRIPTION
**Pull request by lindtdev**
Six landing pages had no content, these have been added descriptions of their content.
Fixes #670

### CODE REVIEW

File: content/kubermatic/master/architecture/_index.en.md [hunk 1]
```diff
@@ -5,4 +5,10 @@ weight = 10
 chapter = true
 +++
 
-# Architecture
+## Architecture
+
+This section contains information about the architecture of KKP clusters, such as the hardware requirements for the master cluster.
```
Comment (zreigz) [inline content/kubermatic/master/architecture/_index.en.md:10]: the text formatting looks strange. Like this:
------------
   -----
     --

File: content/kubermatic/master/tutorials_howtos/_index.en.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ weight = 20
 chapter = true
 +++
 
-# Tutorials & How-tos 
+## Tutorials & How-tos 
 
+This section contains `tutorials` that will show you how to execute basic tasks with your KKP cluster.
```
Comment (zreigz) [inline content/kubermatic/master/tutorials_howtos/_index.en.md:10]: the text alignment looks strange.

File: content/kubermatic/master/architecture/_index.en.md (hunk 1 above)
Comment (lindtdev) [reply content/kubermatic/master/architecture/_index.en.md:7]: Thank you! Fixed it.

**Comment by lindtdev** [comment]
@zreigz the PR is fixed now, can you please take a look again and if everything is okay, merge it?

**Pull request merged**